cli/src/main/java/com/devonfw/tools/ide/tool/az/Azure.java

```diff
@@ -0,0 +1,41 @@
+package com.devonfw.tools.ide.tool.az;
+
+import java.nio.file.Paths;
+import java.util.Set;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.environment.EnvironmentVariables;
+import com.devonfw.tools.ide.environment.EnvironmentVariablesType;
+import com.devonfw.tools.ide.tool.ToolCommandlet;
+
+/**
+ * {@link ToolCommandlet} for azure CLI (azure).
+ */
+
+public class Azure extends ToolCommandlet {
+
+  /**
+   * The constructor.
+   *
+   * @param context the {@link IdeContext}.
+   */
+  public Azure(IdeContext context) {
+
+    super(context, "az", Set.of(TAG_CLOUD));
+  }
+
+  @Override
+  public void postInstall() {
+
+    super.postInstall();
+
+    EnvironmentVariables variables = this.context.getVariables();
+    EnvironmentVariables settingsVariables = variables.getByType(EnvironmentVariablesType.WORKSPACE);
+    // TODO oder EnvironmentVariablesType.CONF
+    // TODO I think there is nothing written to a file, just the env var exported and thereby only active in current
+    // session. Should this env var be written to a file?
```

Comment:
Simply do the change and then call `save` method on the according variables.
FYI: Only `EnvironmentVariablesPropertiesFile` implements `save` properly. So currently doing `variables.save()` would have no effect. Maybe we should throw an `UnsupportedOperationException` in this case so the developer does not get mistaken...

Reply:
See also #78